packages/api/src/Base.ts

```diff
@@ -229,7 +250,7 @@ export default abstract class ApiBase<ApiType> {
    * api.queryMulti(
    *   [
    *     // you can include the storage without any parameters
-   *     api.query.balances.existentialDeposit,
+   *     api.consts.balances.existentialDeposit,
```

Comment:
This example is wrong, you cannot queryMulti on consts, since you don't need to query